Samples.xcodeproj/project.pbxproj

```diff
@@ -2713,6 +2721,22 @@
 			path = "Set feature layer rendering mode on scene";
 			sourceTree = "<group>";
 		};
+		9520B24B2E13520400B3BEF9 /* Show line of sight between geoelements */ = {
```

Comment:
p.s. Please right click to sort the samples folder so Xcode project cleans itself up.